### PR TITLE
Fix a few things inside wallet

### PR DESCRIPTION
**Pull request by klochowicz**
See commit description for details.

### CODE REVIEW

File: rust/src/wallet.rs [hunk 1]
```diff
@@ -19,23 +18,37 @@ pub const TESTNET_ELECTRUM: &str = "ssl://blockstream.info:993";
 /// Wallet has to be managed by Rust as generics are not support by frb
 static WALLET: Storage<Mutex<Wallet>> = Storage::new();
 
+pub enum Network {
+    Mainnet,
+    Testnet,
+    Regtest,
```
Comment (holzeis) [inline rust/src/wallet.rs:24]: ❓ Why are we adding regtest as enum option, if we do not support it?

**Review by holzeis — APPROVED** [review]
Nice 👍

File: rust/src/wallet.rs (hunk 1 above)
Comment (klochowicz) [reply rust/src/wallet.rs:24]: I was just converting the tests initially when I added it. I anticipate it being useful when testing lightning

**Pull request merged**